src/components/pages/rules/flyouts/ruleEditor/ruleEditor.js

```diff
@@ -147,6 +147,7 @@ export class RuleEditor extends LinkedComponent {
           erroe: undefined
         }
       };
+      logEvent(toRuleDiagnosticsModel('Rule_ApplyClick', requestProps));
```

Comment:
Do we differentiate between create and update? Does ApplyClick is for both?

Reply:
We have an initial click event that is different for edit/update but then all the events after are the same. We could differentiate here too but since the other events don't differentiate I thought it made more sense to keep it the same event

Reply:
That make sense.